readthedocs/proxito/views/serve.py

```diff
@@ -468,7 +452,9 @@ class ServeError404(SettingsOverrideObject):
 
 class ServeRobotsTXTBase(ServeDocsMixin, View):
 
-    @method_decorator(cache_page(60 * 60))  # 1 hour
```

Comment:
Are we telling the CDN how long to cache these? I assume that we're also explicitly clearing these with `Cache-Tags`, but I think we likely want the ability to define duration in some fashion at the CDN level..

Reply:
I was thinking about this as well and I'm assuming that it will come in a following PR since we don't already have the headers set in place for this. IIRC, this is define at Cloudflare and it has the same value for all the pages (2hs, I guess)

Reply:
we have https://github.com/readthedocs/readthedocs.org/issues/10035 open for that.